optimum/habana/transformers/generation/utils.py

```diff
@@ -432,6 +434,11 @@ def generate(
         if streamer is not None:
             streamer.put(input_ids.cpu())
 
+        # determine whether limit_hpu_graphs needs to be used
+        model_kwargs["limit_hpu_graphs"] = (
+            generation_config.limit_hpu_graphs if hasattr(generation_config, "limit_hpu_graphs") else False
+        )
```

Comment:
Let's define `limit_hpu_graphs` in: https://github.com/huggingface/optimum-habana/blob/4bf8a224b8838850e67ac85ca7fdbca91cc09d2d/optimum/habana/transformers/generation/configuration_utils.py#L19
so that we don't have to check here if it's in the generation config.